data/json/items/armor.json

```diff
@@ -12948,6 +12948,34 @@
         "flags" : ["VARSIZE", "OVERSIZE", "STURDY"],
         "coverage" : 75,
         "material_thickness" : 3
-    }
+    },
+    {
+	"type" : "ARMOR",
+	"id" : "jerrypack",
+	"name" : "jerrypack",
+	"weight" : 1763,
+	"color" : "green",
+	"covers" : ["TORSO"],
+	"to_hit" : -1,
+	"storage" : 0,
+	"symbol" : "[",
+	"description" : "A homemade container made by nailing some rags to the side of a jerrycan. It is useful in simplifying the hauling of liquids.",
+	"price" : 1450,
+	"material" : ["plastic", "cotton"],
+	"volume" : 40,
+	"cutting" : 0,
+	"bashing" : 0,
+	"warmth" : 5,
+	"environmental_protection" : 0,
+	"encumbrance" : 30,
+	"flags" : ["VARSIZE", "BELTED"],
```

Comment:
That "VARSIZE" doesn't really fit. It's a jerrycan with straps, it shouldn't be refitable.
Did you mean OVERSIZE?